Multiple databases (#779)
timescale/docs: cloud/customize-configuration.md

and the `Apply changes` (or `Restart and apply changes`) button is available to
click, prompting you to confirm any changes before the Service is modified.

## Multiple databases

> Today, Timescale Cloud does not allow the creation of multiple databases within 
> a single service. If you require multiple databases, you will need to create a 
> service for each database. Another option would be to redesign your table 
> structure. Timescale Cloud does not support multiple databases within the same 
> service to reduce complexity. Each database would require its own installation 
> of TimescaleDB, managed independently, which breaks our ability to manage 
> maintenance upgrades for you. Supporting multiple databases also adds 
> significant complexity to the UI. To keep our services as easy to use as 
> possible, we have opted to limit services to one database per service.  

Reviewer: ```suggestion
To create more than one Timescale Cloud database, you need to create a new 
service for each database. Timescale Cloud does not support multiple 
databases within the same service. 
```

Reviewer: I would really like to have some actual instructions for "redesign your table structure", because that's not really descriptive enough. I'm also not convinced we need all the internal reasons for designing it this way, it reads a little like "we found this difficult, so we didn't do it".

Reviewer: Additionally, if the content is really just "use one service per db" then it should be an admonition, not under its own heading.

Author: Incorporated these changes into my next commit! Let me know if you feel this isn't sufficient. I agree that I am not sure that this warrants its own section.

and the `Apply changes` (or `Restart and apply changes`) button is available to
click, prompting you to confirm any changes before the Service is modified.

## Multiple databases
To create more than one Timescale Cloud database, you need to create a new 
service for each database. Timescale Cloud does not support multiple 
databases within the same service. Having a separate service for each database 
affords each database its own isolated resources. 

> Another option is to use [schemas](https://www.postgresql.org/docs/current/ddl-schemas.html). Schemas provide a way to organize 
> tables into logical groups. A single database can contain multiple schemas, 
> which in turn contain tables. The main difference between isolating with 
> databases versus schemas is that a user can access objects in any of the 
> schemas in the database they are connected to, so long as they have the 
> corresponding privileges. Schemas can help isolate smaller use cases 
> that do not warrant their own service.

Reviewer: ```suggestion
Another option is to use 
[schemas](https://www.postgresql.org/docs/current/ddl-schemas.html). 
Schemas provide a way to organize tables into logical groups. A single 
database can contain multiple schemas, which in turn contain tables. 
The main difference between isolating with databases versus schemas 
is that a user can access objects in any of the schemas in the database 
they are connected to, so long as they have the corresponding privileges. 
Schemas can help isolate smaller use cases that do not warrant their 
own service.
```

Reviewer: Instead of all this info about what a schema is, why not just link out to our own Schema how to? (I note you've linked to the pg docs, not internally). Then, you don't need to explain what schemas are, necessarily, you can just say something like "if you want to run multiple datasets within a single service, you can use schemas within a single database. For more information about schemas, see the [Schema documentation](https://docs.timescale.com/timescaledb/latest/how-to-guides/schema-management/)" (or similar).

and the `Apply changes` (or `Restart and apply changes`) button is available to
click, prompting you to confirm any changes before the Service is modified.

## Multiple databases
To create more than one Timescale Cloud database, you need to create a new 
service for each database. Timescale Cloud does not support multiple 
databases within the same service. Having a separate service for each database 
affords each database its own isolated resources. 

Another option is to use 
[schemas](https://www.postgresql.org/docs/current/ddl-schemas.html). 
Schemas provide a way to organize tables into logical groups. A single 
database can contain multiple schemas, which in turn contain tables. 
The main difference between isolating with databases versus schemas 
is that a user can access objects in any of the schemas in the database 
they are connected to, so long as they have the corresponding privileges. 
Schemas can help isolate smaller use cases that do not warrant their 

> own service.

Author: Sorry, I don't follow. Aren't our docs only talking about schemas in the other sense of the word? I don't think we have docs on schemas in this sense.

Apologies if I am missing something

> own service.

Reviewer: In that case, we should probably add it 😉
